### PR TITLE
ISSD-1243 [Marketplace] Change from headless admin to delivery in Solutions Marketplace page

### PR DESCRIPTION
**Pull request by juliocezar96**
Subtask : [ISSD-1243](https://liferay.atlassian.net/browse/ISSD-1243)

### CODE REVIEW

**Comment by juliocezar96** [comment]
## :fire: GitRay Report 
Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.
 ## Jira Tasks 

 <html></html> 

---

Hey, don't forget to update all related tickets with correct status, such as **In Review** :wink: 
### Git Stats 

1 file changed, 24 insertions(+), 9 deletions(-)

**Comment by juliocezar96** [comment]
ci:test:sf

**Review by kevenleone — APPROVED** [review]
LGTM 🔥 

@juliocezar96 SF please

**Comment by juliocezar96** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/solutions-search-results/ddm-template.ftl [hunk 1]
```diff
@@ -85,6 +85,16 @@
 	</div>
 </#if>
 
+<#if themeDisplay?has_content>
+	<#assign scopeGroupId = themeDisplay.getScopeGroupId() />
+</#if>
+
+<#assign channel = restClient.get("/headless-commerce-delivery-catalog/v1.0/channels?accountId=-1&filter=name eq 'Marketplace Channel' and siteGroupId eq '${scopeGroupId}' ") />
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/solutions-search-results/ddm-template.ftl:92]: Please, remove this white space.
```suggestion
<#assign channel = restClient.get("/headless-commerce-delivery-catalog/v1.0/channels?accountId=-1&filter=name eq 'Marketplace Channel' and siteGroupId eq '${scopeGroupId}'") />
```

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/solutions-search-results/ddm-template.ftl [hunk 2]
```diff
@@ -123,7 +141,7 @@
 							<img
 								alt=${productName}
 								class="solution-search-image rounded"
-								src="${productThumbnail}"
+								src="/o/${productImageParts[1]}"
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/solutions-search-results/ddm-template.ftl:144]: What will happen if the productImageParts to be null or undefined?

**Review by silasbsi — CHANGES_REQUESTED** [review]
I added some suggestions

**Comment by juliocezar96** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/solutions-search-results/ddm-template.ftl (hunk 2 above)
Comment (silasbsi) [reply modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/solutions-search-results/ddm-template.ftl:143]: Where was productImageParts defined?

**Review by silasbsi — CHANGES_REQUESTED** [review]
I added a new question

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/solutions-search-results/ddm-template.ftl (hunk 2 above)
Comment (juliocezar96) [reply modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/solutions-search-results/ddm-template.ftl:143]: OMG. I forgot to update this line

Comment (juliocezar96) [reply modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/solutions-search-results/ddm-template.ftl]: And if productImageParts is null or undefined, this is put an image default there is in the portal. I took a test, where I took images of a product and it placed this default image.

**Review by claricebrasil — APPROVED** [review]
Good job, Julio!

**Comment by juliocezar96** [comment]
ci:test:sf

**Comment by juliocezar96** [comment]
ci:test:relevant

**Review by silasbsi — APPROVED** [review]
Good job @juliocezar96

**Comment by silasbsi** [comment]
ci:forward:force

**Comment by silasbsi** [comment]
Just starting reviewing :)

**Comment by silasbsi** [comment]
Resend: https://github.com/brianchandotcom/liferay-portal/pull/142715

**Pull request closed without merging**